drivers/adc/ad9656/ad9656.c

```diff
@@ -0,0 +1,202 @@
+/***************************************************************************//**
```

Comment:
The modifications that were done must be included in a changelog in the pull request something like:
```
Changelog v2:
- added Signed-off-by tag
- check return value
```
The commit description must contain in this case some details of the chip and some capabilities that the driver supports.

Reply:
The commit still does not describe nothing about the chip. 
```
Modified the code according to the pull request comments.
```
Is not telling any information about the implementation and has no value.

Reply:
Changelog v2:

- checked the return value of the spi_remove operation
- modified the call for the functions to be written in one line
- modified if clause to respect the Linux coding style
- deleted the unnecessary initialization of return variable
- restructured the code for the JESD204 test function
- modified the commenting style from // to /* */
- modified the copyright year
- checked if the read SPI function was successful
- deleted unnecessary return variable and called the spi_write_and_read directly from return
- used struct clause when declaring a spi_init_param variable
- used struct clause when declaring a spi_desc variable
- replaced the include of "platform_drivers.h" with "spi.h"
- modified and completed the function comments
- checked return value for spi_init function